components/component_style/navbar.css

```diff
@@ -60,3 +60,34 @@
 }
 
 
+.page {
+  opacity: 0.85;
+  font-weight: 600;
+}
+
+.page:hover {
+  color: var(--primary-color);
+  opacity: 1!important;
```

Comment:
Suggestion: Avoid the use of `!important` try to solve the problem using specificity maybe.